stock_release_channel/models/stock_move.py

```
@@ -25,12 +25,24 @@ def release_available_to_promise(self):
                 moves.picking_id.assign_release_channel()
         return res
 
-    def _action_confirm(self, merge=True, merge_into=False):
-        moves = super()._action_confirm(merge=merge, merge_into=merge_into)
-        pickings = moves.filtered("need_release").picking_id
-        if pickings:
-            pickings._delay_assign_release_channel()
-        return moves
+    def _unreleased_to_backorder(self, split_order=False):
```

This PR is missing https://github.com/OCA/stock-logistics-reservation/pull/38. It should be added to the `test-requirements.txt` file, until it is not merged. 

Also it seems like there is not test which is hitting this part otherwise your tests should raise an error, because `_unreleased_to_backorder` until does not allow any additional parameters.

---

It would be nice to have a one test here.

---

Thanks for the review.
I can skip the pending PR as there is a default value.
I added a test to cover to check the backorder as a job to assign a channel. It was tested in https://github.com/OCA/stock-logistics-release-channel/pull/38 but it's good to have a test here as well.

---

You were right, now the test is failing due to the missing dependency :)